src/snowflake/cli/plugins/snowpark/venv.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import subprocess
+import venv
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+log = logging.getLogger(__name__)
+
+
+class Venv(object):
```

Comment:
No need to inherit from `object`

```suggestion
class Venv:
```

Reply:
Done